### ISSUE TITLE
No values available for perfevent.hwcounters.cpu.cpu_cycles.value

### ISSUE THREAD

**Issue by pradykaushik** [performancecopilot/pcp#1158]
**software specs**
1. PCP version - 4.0.1
2. OS - Ubuntu 18.04.1 LTS

**hardware specs**
1. Processor - Intel(R) Xeon(R) CPU E5-2620 v3 @ 2.40GHz 
2. \#CPUs = 24 (2 sockets * 6 cores each with hyper-threading enabled).

**perfevent.conf file dynamic metrics**
Below are the dynamic metrics included in /var/lib/pcp/pmdas/perfevent/perfevent.conf.
```
[dynamic]
cpu.cpu_cycles
cpu.instructions
cpu.branch-misses
cpu.cache-references
software.page-faults
software.context-switches
```

## Issue
I am able to retrieve the number of instructions executed. However, I'm not able to retrieve the number of cpu cycles and I get an **Unknown metric name** error message.

**Command** - `pmdumptext -m -l -f '' -t 1.0 -d , perfevent.hwcounters.cpu.instructions.value`

**Output**:
```commandline
local::perfevent.hwcounters.cpu.instructions.value["cpu0"],local::perfevent.hwcounters.cpu.instructions.value["cpu1"],local::perfevent.hwcounters.cpu.instructions.value["cpu2"],local::perfevent.hwcounters.cpu.instructions.value["cpu3"],local::perfevent.hwcounters.cpu.instructions.value["cpu4"],local::perfevent.hwcounters.cpu.instructions.value["cpu5"],local::perfevent.hwcounters.cpu.instructions.value["cpu6"],local::perfevent.hwcounters.cpu.instructions.value["cpu7"],local::perfevent.hwcounters.cpu.instructions.value["cpu8"],local::perfevent.hwcounters.cpu.instructions.value["cpu9"],local::perfevent.hwcounters.cpu.instructions.value["cpu10"],local::perfevent.hwcounters.cpu.instructions.value["cpu11"],local::perfevent.hwcounters.cpu.instructions.value["cpu12"],local::perfevent.hwcounters.cpu.instructions.value["cpu13"],local::perfevent.hwcounters.cpu.instructions.value["cpu14"],local::perfevent.hwcounters.cpu.instructions.value["cpu15"],local::perfevent.hwcounters.cpu.instructions.value["cpu16"],local::perfevent.hwcounters.cpu.instructions.value["cpu17"],local::perfevent.hwcounters.cpu.instructions.value["cpu18"],local::perfevent.hwcounters.cpu.instructions.value["cpu19"],local::perfevent.hwcounters.cpu.instructions.value["cpu20"],local::perfevent.hwcounters.cpu.instructions.value["cpu21"],local::perfevent.hwcounters.cpu.instructions.value["cpu22"],local::perfevent.hwcounters.cpu.instructions.value["cpu23"]
?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?
1725514.251,114382.643,490776.819,216932.979,2183895.703,335286.054,3807153.577,809814.197,378284.905,54142.258,335538.018,1244609.021,228010.395,150631.460,80621.471,56049.985,408051.649,75050.268,240418.620,494452.293,9094385.503,210716647.519,2058616.618,1003277.531
...
```

**Command** - `pmdumptext -m -l -f '' -t 1.0 -d , perfevent.hwcounters.cpu.cpu_cycles.value`

**Output**:
```commandline
pmdumptext: no valid metrics, exiting.
```

What could be the reason for this?

**Comment by natoscott** [comment]
Any clues in /var/log/pcp/pmcd/perfevent.log ?

There's been alot of changes in pmdaperfevent since pcp-4.0.1 FWIW, might be worth trying latest version too.

**Comment by pradykaushik** [comment]
The newest logs in that file are from June 2020.
I'm going to try upgrading PCP to the latest version available (5.2.2) and will get back to you here. Thanks.

**Comment by natoscott** [comment]
Closing for now - please re-open if the upgrade doesn't help @pradykaushik
